<!DOCTYPE html>
<html>
<head>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-58482298-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-58482298-2');
</script>



    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Projects</title>
    <meta name="description" content="LiSyM Junior Group König
">


    <link rel="stylesheet" href="/css/main.css">
    <link rel="stylesheet" href="/css/font-awesome.min.css">

    <link rel="shortcut icon" href="/favicon.ico?1">
    <!-- Begin Jekyll SEO tag v2.2.2 -->
<title>Projects | livermetabolism</title>
<meta property="og:title" content="Projects" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="LiSyM Junior Group König" />
<meta property="og:description" content="LiSyM Junior Group König" />
<link rel="canonical" href="http://localhost:4000/projects/" />
<meta property="og:url" content="http://localhost:4000/projects/" />
<meta property="og:site_name" content="livermetabolism" />
<script type="application/ld+json">
{"@context":"http://schema.org","@type":"WebPage","headline":"Projects","description":"LiSyM Junior Group König","url":"http://localhost:4000/projects/"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="canonical" href="http://localhost:4000/projects/">
    <link rel="alternate" type="application/rss+xml" title="livermetabolism" href="http://localhost:4000/feed.xml" />


</head>
<body>

    <nav class="navbar navbar-default navbar-fixed-top">
    <div class="container navbar-container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
            <a class="navbar-brand" href="/">
                <span><img src="/img/logonav.png"></span> livermetabolism
            </a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
                <li ><a href="/projects/">Projects</a></li>
                <li ><a href="/publications/">Publications</a></li>
                <li ><a href="/people/">People</a></li>
                <li ><a href="/positions/">Positions</a></li>
                <li ><a href="/cv/">CV</a></li>
            </ul>
            <div class="navbar-right">
                <ul class="nav navbar-nav">
                    <li><a href="https://github.com/matthiaskoenig"><i class="fa fa-github" aria-hidden="true"></i></a></li>
                </ul>
            </div>
        </div>
    </div>
</nav>


    <div class="page-content">
        <div class="wrapper">
            <div class="post">

    <header class="post-header">
        <h1 class="post-title">Projects</h1>
    </header>

    <article class="post-content">
        <div class="container">
    
    <div class="row">
        <h2> Multi-scale Model of Hepatic Metabolism</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                </td>
                <td width="150">
                    


                        

                        

                        
                        <a href="../paper/Multiscale.galactose_Koenig2018.pdf"><img src="/img/pdf.png" title="pdf" height="30"/></a>
                        

                    
                </td>
                <td>
                    
                    <b>König M.</b> et al.<br/>
                    <i>A Multiscale Computational Model Predicts Human Liver Function From Single-Cell Metabolism</i><br/>
                    [in preparation]
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="Multi-scale Model of Hepatic Metabolism" title="Multi-scale Model of Hepatic Metabolism" src="/img/projects/multiscale-galactose.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>Multiscale model of human galactose metabolism - from single hepatocytes to individual liver function.</h3> <p> Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. </p> <!-- <p> Calculation of individual liver function based on Galactose Elimination Capacity (GEC). </p> <p> <a href="https://livermetabolism.com/gec_app" target="_blank"> <img src="/img/projects/gec_app.png" /> </a> --> </p>
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> Liver Function Tests</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                    
                    
                    <img src="/img/people/eleftheriadou.png" class="img-circle" height="50" title="Dimitra Eleftheriadou"/>
                    
                    
                </td>
                <td width="150">
                    
                        
                        <a href="https://github.com/matthiaskoenig/"><img src="/img/project.png" title="project homepage"
                                                             height="30"/></a>
                        
                        
                        <a href="https://github.com/matthiaskoenig/"><img src="/img/repository.png" title="repository"
                                                                height="30"/></a>
                        
                    
                </td>
                <td>
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="Liver Function Tests" title="Liver Function Tests" src="/img/projects/function-tests.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                    <h3>Personalized evaluation of liver function tests</h3> <p> Understanding how liver function results from the complex interaction of hepatic morphology, perfusion and metabolism across multiple spatial scales, from the cellular level up to the entire organ, requires systems levels computational approaches. Despite the importance of quantifying liver function in the clinics, currently no computational model of the human liver is able to utilize the morphological and physiological information from imaging methods for a personalized evaluation of quantitative function tests.
Our objectives are <ul> <li>providing an open, freely available and reproducible multiscale model of the human liver which can readily be employed to evaluate the hepatic clearance of various test substances used in quantitative liver function tests in the clinics</li>
<li>providing the infrastructure and methods to integrate experimental and clinical data on all scales (imaging data, anthropomorphic information, perfusion data multipleindicator dilution data, microcirculation and ultrastructure parameters, protein amounts, ...)</li>
<li>create metabolic modules for the elimination of various test substances and drugs (methacetin, galactose, BSP, ICG, midazolam, codeine, caffeine, torsemide, pravastatin, talinolol) by the liver. This will allow the analysis of a wide range of clinical studies and create an integrative platform for the clearance of these substances by the liver.</li> </ul> </p>
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> Biomarker signatures in liver disease</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                </td>
                <td width="150">
                    


                        
                        <a href="http://matthiaskoenig.github.io/bdl-analysis/"><img src="/img/project.png" title="project homepage"
                                                                 height="30"/></a>
                        

                        
                        <a href="http://github.com/matthiaskoenig/bdl-analysis/"><img src="/img/repository.png" title="repository"
                                                                    height="30"/></a>
                        

                        
                        <a href="../paper/Pathobiochemical.signatures_Abshagen2015.pdf"><img src="/img/pdf.png" title="pdf" height="30"/></a>
                        

                    
                </td>
                <td>
                    
                    Kerstin A.<b>*</b>, <b>König M.*</b>, Hoppe A., Thomas M., Müller I., Ebert M., Weng H., Holzhütter HG., Zanger UM., Bode J., Vollmar B. and Dooley S.<br /><b>* equal contribution</b><br/>
                    <i>Pathobiochemical signatures of cholestatic liver disease in bile duct ligated mice</i><br/>
                    BMC Syst Biol. 2015 Nov 20;9(1):83. doi: 10.1186/s12918-015-0229-0 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26589287">PubMed</a>]
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="Biomarker signatures in liver disease" title="Biomarker signatures in liver disease" src="/img/projects/BDL_correlation_heatmap.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>Pathobiochemical signatures of cholestatic liver disease</h3> <p> Disrupted bile secretion leads to liver damage characterized by inflammation, fibrosis, eventually cirrhosis, and hepatocellular cancer. As obstructive cholestasis often progresses insidiously, markers for the diagnosis and staging of the disease are urgently needed. To this end, we compiled a comprehensive data set of serum markers, histological parameters and transcript profiles at 8 time points of disease progression after bile duct ligation (BDL) in mice, aiming at identifying a set of parameters that could be used as robust biomarkers for transition of different disease progression phases. </p> 
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> High Performance Simulators and Simulation Software</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                </td>
                <td width="150">
                    


                        
                        <a href="http://tellurium.analogmachine.org/"><img src="/img/project.png" title="project homepage"
                                                                 height="30"/></a>
                        

                        
                        <a href="https://github.com/sys-bio/tellurium"><img src="/img/repository.png" title="repository"
                                                                    height="30"/></a>
                        

                        
                        <a href="../paper/Tellurium.Notebook_Medley2017.pdf"><img src="/img/pdf.png" title="pdf" height="30"/></a>
                        

                    
                </td>
                <td>
                    
                    Medley K., Choi K., <b>König M.</b>, Smith L., Gu S., Hellerstein J., Sealfon S., and Sauro HM.<br/>
                    <i>Tellurium Notebooks - An Environment for Dynamical Model Development, Reproducibility, and Reuse</i><br/>
                    bioRxiv 239004; doi: <a href="https://doi.org/10.1101/239004">https://doi.org/10.1101/239004</a>
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="High Performance Simulators and Simulation Software" title="High Performance Simulators and Simulation Software" src="/img/projects/tellurium-notebook.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>Tellurium Notebooks - An Environment for Dynamical Model Development, Reproducibility, and Reuse</h3> <p>The considerable difficulty encountered in reproducing the results of published dynamical models limits validation, exploration and reuse of this increasingly large biomedical research resource. To address this problem, we have developed Tellurium Notebook, a software system that facilitates building reproducible dynamical models and reusing models by 1) supporting the COMBINE archive format during model development for capturing model information in an exchangeable format and 2) enabling users to easily simulate and edit public COMBINE-compliant models from public repositories to facilitate studying model dynamics, variants and test cases. Tellurium Notebook, a Python--based Jupyter--like environment, is designed to seamlessly inter-operate with these community standards by automating conversion between COMBINE standards formulations and corresponding in--line, human--readable representations. Thus, Tellurium brings to systems biology the strategy used by other literate notebook systems such as Mathematica. These capabilities allow users to edit every aspect of the standards--compliant models and simulations, run the simulations in--line, and re--export to standard formats. We provide several use cases illustrating the advantages of our approach and how it  allows development and reuse of models without requiring technical knowledge of standards. Adoption of Tellurium should accelerate model development, reproducibility and reuse. </p>
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> Reproducible simulation experiments</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                    
                    
                    <img src="/img/people/grzegorzewski.png" class="img-circle" height="50" title="Jan Grzegorzewski"/>
                    
                    
                </td>
                <td width="150">
                    
                        
                        <a href="https://github.com/matthiaskoenig/tellurium-web/"><img src="/img/project.png" title="project homepage"
                                                             height="30"/></a>
                        
                        
                        <a href="https://github.com/matthiaskoenig/tellurium-web/"><img src="/img/repository.png" title="repository"
                                                                height="30"/></a>
                        
                    
                </td>
                <td>
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="Reproducible simulation experiments" title="Reproducible simulation experiments" src="/img/projects/tellurium-web.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>tellurium-web: An online database for reproducible simulation experiments in computational biology</h3> <p> We present tellurium-web, a free and open online database and simulation tool for reproducible simulation experiments in computational biology. Tellurium-web is build on top of existing standards in computational biology, i.e., the SBML for models, SED-ML for description of simulation experiments and the COMBINE archive format for exchange of simulation studies. Key features are 1) large set of example archives (e.g., simulation studies from biomodels, SED-ML specification examples, SBML testsuite, JWS online, tellurium examples); 2) COMBINE archive browser with access to metadata and content of the archives; 3) Execution of archives using a high-performance simulator (libroadrunner) integrated into modeling platform for computational biology (tellurium); 4) REST web services for getting, posting and modifying archives providing means for simple integration into computational workflows; and  4) filtering of archives based on tags and users. </p>
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> FluType: Influenza Classification based on Machine Learning</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/grzegorzewski.png" class="img-circle" height="50" title="Jan Grzegorzewski"/>
                    
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                </td>
                <td width="150">
                    
                        
                        <a href="https://www.flutype.de/about/"><img src="/img/project.png" title="project homepage"
                                                             height="30"/></a>
                        
                        
                        <a href="https://github.com/janekg89/flutype_webapp"><img src="/img/repository.png" title="repository"
                                                                height="30"/></a>
                        
                    
                </td>
                <td>
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="FluType: Influenza Classification based on Machine Learning" title="FluType: Influenza Classification based on Machine Learning" src="/img/projects/flutype.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>Development of an influenza sub-classification platform based on peptides</h3> <p> The FluTypeDB project is a web application for the data management of binding assays for the classification of influenza subtypes. FluTypeDB consists of a database and web interface with focus on various binding assays for the classification of influenza viruses and contains experimental data based on <ul> <li>ELISA antibody binding assays</li> <li>Peptid binding assays on microwell plates</li> <li>Peptid bindings assays on peptide microarrays</li> </ul> </p>
                </p>
            </div>
        </div>
    </div>
    
    <div class="row">
        <h2> Visualization of kinetic data for modeling</h2>

        

        <table class="table table-responsive well">
            <tr>
                <td width="150">
                    
                    
                    
                    <img src="/img/people/koenig.png" class="img-circle" height="50" title="Dr. Matthias König"/>
                    
                    
                </td>
                <td width="150">
                    


                        
                        <a href="http://apps.cytoscape.org/apps/cy3sabiork"><img src="/img/project.png" title="project homepage"
                                                                 height="30"/></a>
                        

                        
                        <a href="http://www.github.com/matthiaskoenig/cy3sabiork"><img src="/img/repository.png" title="repository"
                                                                    height="30"/></a>
                        

                        
                        <a href="../paper/cy3sabiork_Koenig2016.pdf"><img src="/img/pdf.png" title="pdf" height="30"/></a>
                        

                    
                </td>
                <td>
                    
                    <b>König M.</b><br/>
                    <i>cy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK</i><br/>
                    [version 1; referees: awaiting peer review]. F1000Research 2016, 5:1736 (doi: <a href="http://dx.doi.org/10.12688/f1000research.9211.1">10.12688/f1000research.9211.1</a>)
                    
                </td>
            </tr>
        </table>
        <div class="row">
            <div class="col-md-6 text-justify">
                <p>
                    
                    <img alt="Visualization of kinetic data for modeling" title="Visualization of kinetic data for modeling" src="/img/projects/cy3sabiork-workflow.png"
                         class="img-responsive"/>
                    
                </p>
            </div>
            <div class="col-md-6 text-justify">
                <p>
                     <h3>Cytoscape app for visualizing kinetic data from SABIO-RK</h3> <p> Kinetic data of biochemical reactions are essential for the creation of kinetic models of biochemical networks. One of the main resources of such information is SABIO-RK, a curated database for kinetic data of biochemical reactions and their related information. Despite the importance for computational modelling there has been no simple solution to visualize the kinetic data from SABIO-RK. In this work, I present cy3sabiork, an app for querying and visualization of kinetic data from SABIO-RK in Cytoscape. The kinetic information is accessible via a combination of graph structure and annotations of nodes, with provided information consisting of: (I) reaction details, enzyme and organism; (II) kinetic law, formula, parameters; (III) experimental conditions; (IV) publication; (V) additional annotations. cy3sabiork creates an intuitive visualization of kinetic entries in form of a species-reaction-kinetics graph, which reflects the reaction-centered approach of  SABIO-RK. Kinetic entries can be imported in SBML format from either the SABIO-RK web interface or via web service queries. The app allows for easy comparison of kinetic data, visual inspection of the elements involved in the kinetic record and simple access to the annotation information of the kinetic record. The cy3sabiork approach was applied in the computational modelling of galactose metabolism in the human liver. </p>
                </p>
            </div>
        </div>
    </div>
    

    <h2>More</h2>
    For more projects see our <strong><a href="/publications/"><i
        class="fa fa-newspaper-o" aria-hidden="true"></i> Publications</a></strong>.

</div>

    </article>

</div>

        </div>
    </div>

    <footer class="footer">
    <div class="container">

        <p class="text-center">
            livermetabolism 2018
        </p>
        <!-- <p class="text-muted">Place sticky footer content here.</p> -->
    </div>
</footer>

    <script>
  var baseurl = ''
</script>
<script src="//code.jquery.com/jquery-1.10.2.min.js"></script>
<script src="/js/bootstrap.min.js "></script>
<script src="/js/typeahead.bundle.min.js "></script>

<script src="/js/main.js "></script>

</body>

</html>
